CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 4.3.2
+**Widget changes:**
+Remove custom worker initializer to avoid forcing apps to remove the default one in the manifest or to avoid crashes when the app has its own custom initializer
```

Comment:
```suggestion
- Remove custom worker initializer to avoid forcing apps to remove the default one in the manifest or to avoid crashes when the app has its own custom initializer
```